scripts/coreos-enable-fips

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -euo pipefail
+
+RPM_OSTREE_BIN=$(which rpm-ostree)
+
+
+# check_and_enable looks to see if the kernel arg is already there. If it is not
+# there, the argument will be added
+function check_and_enable() {
+    CHECK_FOR_ARG=$1
+    # Add the kernel arg if it does not exist
+    set +e
+    ${RPM_OSTREE_BIN} kargs | grep "${CHECK_FOR_ARG}" > /dev/null
+    ARG_CHECK=$?
+    set -e
+    if [[ ${ARG_CHECK} != 0 ]]; then
+        ${RPM_OSTREE_BIN} kargs --append=${CHECK_FOR_ARG}
+    fi
+}
+
+# is_ostree_system checks /etc/os-release for ostree variables
+# in an attempt to verify this is an ostree owned system
+function is_ostree_system() {
```

Comment:
`/run/ostree-booted` is the more canonical way of checking if a host is `ostree` based

Reply:
Updated